packages/multichain-network-controller/CHANGELOG.md

```diff
@@ -7,14 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
 ### Added
 
 - Testnet asset IDs added as constants ([#5589](https://github.com/MetaMask/core/pull/5589))
 - Network specific decimal values and ticker as constants ([#5589](https://github.com/MetaMask/core/pull/5589))
+- Acts as a proxy to remove an EVM network from the `@metamask/network-controller` ([#5516](https://github.com/MetaMask/core/pull/5516))
```

Comment:
I think something is missing here maybe you want to reference the new method `removeNetwork `

Reply:
How does this work? I was unsure why the error was being triggered so I just added the label assuming it was not related

Reply:
Will add the method as suggested